### ISSUE TITLE
Shuffle sometimes injects undefined into array

### ISSUE THREAD

**Issue by BryanDonovan** [Marak/faker.js#175]
This test fails in helpers.unit.js:

```javascript
    describe("shuffle()", function () {
        it("returns an array of the correct size", function () {
            var arr = ['a', 'b', 'c'];
            for (var i = 0; i < 1000; i++) {
                var shuffled = faker.helpers.shuffle(arr);
                assert.strictEqual(shuffled.length, 3);
            }
        });
    });
```
It sometimes returns an array with an `undefined` element, like: `[ 'b', 'a', undefined, 'c' ]`.

**Comment by rubenstolk** [comment]
@BryanDonovan do you mind adding your test to this PR, so we can get it merged:

https://github.com/Marak/faker.js/pull/160

I really don't understand why this PR is still open and why nobody faced this issue before...

**Comment by rubenstolk** [comment]
Issue is related to https://github.com/Marak/faker.js/issues/159

**Comment by BryanDonovan** [comment]
Sure, I'll add it.

On a side note, it's a shame that running `make` fails so badly now.  Sigh.

**Comment by Marak** [comment]
Fixed with `v2.1.4`
